tests/unit/inference/test_inference.py

```diff
@@ -298,6 +298,14 @@ def verify_injection(module):
     verify_injection(model)
 
 
+# Used to Get Device name
+def getDeviceId(local_rank):
+    device = local_rank
+    if get_accelerator().device_name() != 'cuda':
```

Comment:
I think either `get_accelerator().device(locak_rank)` or `get_accelerator().device_name(local_rank)` should used here for all accelerators including `cuda`. Did you run into problems with those?

Reply:
Updated as per comment. We faced issue when we used local rank for accelerator so we modified device as per name but kept the cuda to be same as local_rank. 
Raised new PR https://github.com/microsoft/DeepSpeed/pull/6397 for the same.

Reply:
Thanks for applying the proper fix. I approved #6397.